scripts/automation/pistonbolt.arucas

```diff
@@ -680,171 +1185,1429 @@ class CyclicTaskGenerator {
+new LinearCommand([['eDiagonalBolt'], ['x', 'Integer', rayTraceX],['y', 'Integer', rayTraceY],['z', 'Integer', rayTraceZ], ['direction', 'Word', ['north', 'west', 'east', 'south']]], diagonalBoltScheduledTask).process();
+new LinearCommand([['eCommand'], ['setReachDistance'], ['reachDistance', 'Double']], reachDistanceScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['setItem'], ['itemName', 'Word', ['piston', 'sticky_piston']]], setPistonItemScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['x', 'Integer', rayTraceX],['y', 'Integer', rayTraceY],['z', 'Integer', rayTraceZ], ['headless', 'Boolean']], breakingScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['setPos1'] ,['x', 'Integer', rayTraceX],['y', 'Integer', rayTraceY],['z', 'Integer', rayTraceZ]], setAreaPos1ScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['setPos1'] ,['rayTrace']], setAreaPos1RayTraceScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['setPos2'] ,['x', 'Integer', rayTraceX],['y', 'Integer', rayTraceY],['z', 'Integer', rayTraceZ]], setAreaPos2ScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['setPos2'] ,['rayTrace']], setAreaPos2RayTraceScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['x', 'Integer', rayTraceX],['y', 'Integer', rayTraceY],['z', 'Integer', rayTraceZ], ['direction', 'Word', ['north', 'west', 'east', 'south', 'up', 'down']], ['headless', 'Boolean']], headlessScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['rayTrace']], breakingLookingAtScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],  ['rayTrace'], ['direction', 'Word', ['north', 'west', 'east', 'south', 'up', 'down']], ['headless', 'Boolean']], breakingLookingAtHeadlessScheduledTask).process();
+new LinearCommand([['eBedrockBreaking'],['Area'], ['x', 'Integer', rayTraceX],['y', 'Integer', rayTraceY],['z', 'Integer', rayTraceZ],['ex', 'Integer', rayTraceX],['ey', 'Integer', rayTraceY],['ez', 'Integer', rayTraceZ]], breakBedrockAreaScheduledTask).process();
+
+showHelp();
 while (true){
-    t.execute();
-}
\ No newline at end of file
+    if (pause){
+        sleep(50);
+        continue;
+    }
+    if (player.getGamemode() == 'spectator'){
+        sleep(50);
+        continue;
+    }
+    leftovers.process();
+    polled = delayedHolder.poll();
+    if (polled != null){
+        taskHolder.add(polled);
+    }
+    foreach (tasks : taskHolder){
+        result = tasks.execute();
+        if (result){
+            taskHolder.remove(tasks);
+        }
+        if (sleepInterval > 0){
+            sleep(sleepInterval);
+        }
+    }
+    foreach (functions : taskFunctionHolder){
+        retVal = functions();
+    }
+}
+
+
```

Comment:
Like this is unreadable. Why is everything in one place. I will merge it but only because I made the mistake of merging it before. In the future please not this. And I beg that you don't add any more features kek

Also file name should follow Pascal casing